### PR TITLE
[Website_blog]: In the paragraph comment system public user should able to comment after login make option For sign In

### PR DESCRIPTION
**Pull request by mba-odoo**
Task: https://www.odoo.com/web#id=10489&view_type=form&model=project.task&menu_id=3942&action=327
Pad: https://pad.odoo.com/p/r.q05FGMJxOeKDthmc

### CODE REVIEW

File: addons/website_blog/static/src/js/website_blog.inline.discussion.js [hunk 1]
```diff
@@ -162,36 +163,13 @@ var InlineDiscussion = core.Class.extend({
...
-                else
-                    $('div#inline_comment').removeClass('has-error');
-                return false;
-            }
-        }
-        else if(!comment) {
+        if(!comment) {
             $('div#inline_comment').addClass('has-error');
-            return false;
+            return false
```
Comment (Gorash) [inline addons/website_blog/static/src/js/website_blog.inline.discussion.js]: Please add ; at the end

File: addons/website_blog/static/src/js/website_blog.inline.discussion.js [hunk 2]
```diff
@@ -162,36 +163,13 @@ var InlineDiscussion = core.Class.extend({
...
         }
         $("div#inline_comment").removeClass('has-error');
-        $('div#author_name').removeClass('has-error');
-        $('div#author_email').removeClass('has-error');
         $(".popover textarea#inline_comment").val('');
-        $('.popover input#author_name').val('');
-        $('.popover input#author_email').val('');
-        return [comment, author_name, author_email];
+        return [comment]
```
Comment (Gorash) [inline addons/website_blog/static/src/js/website_blog.inline.discussion.js]: Please add ; at the end

Comment (mba-odoo) [inline addons/website_blog/static/src/js/website_blog.inline.discussion.js]: done

**Comment by Gorash** [comment]
@tde-banana-odoo

**Comment by Gorash** [comment]
merge: 7278d277d9c0a513342b558bf97e1d0470635df6

**Pull request closed without merging**